assets/js/vendor/cards-expandable.js

```diff
@@ -0,0 +1,12 @@
+// JS for expandable cards (.va-card--expands)
+$(document).ready(function() {
+	$('.va-card-body').on('click', function(e){
+    	if(e.target.nodeName == 'DIV') {
+    		$(e.currentTarget).parent().toggleClass('va-card--hasdrawer--open');
+    	}
+    	// If the drawer is open, disable links in the card's body
+    	if(e.target.nodeName == 'A' && $(e.currentTarget).parent().hasClass('va-card--hasdrawer--open')) {
```

Comment:
@akainic I agree with that, tbh, but that's not what the original design was. 

@gnakm Can you weigh in here? It's an easy change to make, and would simplify the JavaScript a bit.